src/System.Runtime.Analyzers/Core/CallGCSuppressFinalizeCorrectly.cs

```diff
@@ -27,42 +29,239 @@ public abstract class CallGCSuppressFinalizeCorrectlyAnalyzer : DiagnosticAnalyz
                                                                              s_localizableMessageNotCalledWithFinalizer,
                                                                              DiagnosticCategory.Performance,
```

Comment:
This is an FXCop Usage rule. @srivatsn @nguerrera did we decide to make it a performance rule?